config.json.example

```diff
@@ -88,4 +88,5 @@
   "REDIS_PORT": "1234",
   "REDIS_PASSWORD": "12345678",
   "TRUSTED_DOMAINS": "localhost,habitica.com"
+  "NO_CLUSTER": "false"
```

Comment:
Could you remove all now not needed changes from the PR?

Reply:
i'm sorry, i thought was removed